hummingbot/connector/exchange/gate_io/gate_io_active_order_tracker.pyx

```diff
@@ -99,8 +99,8 @@ cdef class GateIoActiveOrderTracker:
 
     cdef tuple c_convert_snapshot_message_to_np_arrays(self, object message):
         cdef:
-            float price
-            float amount
+            object price
+            double amount
```

Comment:
Is there a reason for this change?

Reply:
This change was required to keep the same price values for the diff messages and for the snapshot messages. In lines 63 and 66 of the same file you can notice that those are the types used to process prices and amounts for the diff messages. Without using those same types the snapshots processor was converting prices to floats and adding more decimals. That caused that when an update arrived for that price in a diff message, prices did not match, and the original entry in the OrderBook was not changed.